src/libstd/fs.rs

```diff
@@ -1724,6 +1734,23 @@ mod tests {
         assert!(canary.exists());
     }
 
+    // FIXME(#12795) depends on lstat to work on windows
+    #[cfg(not(windows))]
```

Comment:
Is it really necessary to disable this test on Windows?  #12795 was fixed a long time ago.